zerver/lib/push_notifications.py

```diff
@@ -58,7 +59,7 @@ def __init__(self, user, tokens, alert=None, badge=None, sound=None,
         payload = Payload(alert=alert, badge=badge, sound=sound,
                           category=category, custom=kwargs)
         for token in tokens:
-            data = {'token': token, 'user': user}
+            data = {'token': token, 'email': user.email}
```

Comment:
Maybe this should use `user_id` rather than email?  that seems more stable in the event that we eventually support changing email addresses.